Dockerfile

```diff
@@ -10,7 +10,7 @@ ENV MPV_PACKAGES="\
 ENV FFMPEG_PACKAGES="\
     libavcodec-dev libavformat-dev libavdevice-dev zlib1g-dev"
 ENV BUBBLESUB_PACKAGES="\
-    python3.9 python3.9-dev python3.9-distutils enchant libfftw3-dev xvfb qt5-default"
```

Comment:
Why isn't `enchant` package more necessary? What's different?

Reply:
- The setup is broken (the packages ceased to exist) and I haven't had the time to debug
- We never test it anyway
- It's "optional" e.g. only used by the spell checker and the like

Reply:
I see, we can remove it then